modules/java8/src/main/scala/io/circe/java8/time/TimeInstances.scala

```diff
@@ -36,6 +37,17 @@ trait TimeInstances {
 
   implicit final val encodeInstant: Encoder[Instant] = Encoder.instance(time => Json.fromString(time.toString))
 
+  implicit final val decodeZoneId: Decoder[ZoneId] =
+    Decoder[String].emap{
+      case s if ZoneId.getAvailableZoneIds.contains(s) =>
+        Right(ZoneId.of(s))
+      case s =>
+        Left(s"$s is not an recognized ZoneId.")
```

Comment:
Can you change this message to just be `"ZoneId"` on the model of the others? We're revisiting the role of these messages in 1.0 (see #306), but for now I'd prefer consistency here.